src/unix/core.c

```diff
@@ -835,8 +835,12 @@ void uv__io_start(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
     return;
 #endif
 
-  if (QUEUE_EMPTY(&w->watcher_queue))
+  if (QUEUE_EMPTY(&w->watcher_queue)) {
     QUEUE_INSERT_TAIL(&loop->watcher_queue, &w->watcher_queue);
+    if (loop->on_watcher_queue_updated)
```

Comment:
Style: `loop->on_watcher_queue_updated != NULL` (no implicit coercion to bool)

Happens in a few more places, I won't point them out individually.